README.md

```diff
@@ -7,6 +7,16 @@ skynet-cli is a lightweight CLI to interact with [Skynet](https://siasky.net).
 Skynet is the decentralized CDN and file sharing platform for devs and the
 storage foundation for a Free Internet!
 
+## Installing
+
+If you have [Go](https://golang.org/cmd/go/) installed you can run:
```

Comment:
that gets the repo, don't they still need to run `make release` to install the binaries?

Reply:
Don't think so, i.e. look at the way we install `analyze`:

```
go get gitlab.com/NebulousLabs/analyze
```

Reply:
```
matt@Carl:~/go/src/github.com/NebulousLabs$ go get github.com/NebulousLabs/skynet-cli
package github.com/NebulousLabs/skynet-cli: no Go files in /home/matt/go/src/github.com/NebulousLabs/skynet-cli
```
 `go get` tries to install packages for a projects. I think you need the `-u` and the `/...` to install a new project

```
matt@Carl:~/go/src/github.com/NebulousLabs$ go get -u github.com/NebulousLabs/skynet-cli/...
matt@Carl:~/go/src/github.com/NebulousLabs$ skynet
Use Skynet to upload and share content

Default Skynet Portal: https://siasky.net 

```